service/controller/chart/resource/release/update.go

```diff
@@ -229,6 +229,8 @@ func (r *Resource) newUpdateChange(ctx context.Context, obj, currentState, desir
 			if err != nil {
 				return nil, microerror.Mask(err)
 			}
+			// no-op after rollback
```

Comment:
if we don't quit in here, it actually reach the point of https://github.com/giantswarm/chart-operator/pull/569/files#diff-28b21de142787ce2c87ef523fff62312L249